timon/webclient/webif1/.eslintrc

```diff
@@ -4,7 +4,8 @@
     "html"
   ],
   "rules": {
-    "no-unused-vars": ["error", { "vars": "all", "args": "after-used", "ignoreRestSiblings": false, "varsIgnorePattern": "^_" }]
+    "no-unused-vars": ["error", { "vars": "all", "args": "after-used", "ignoreRestSiblings": false, "varsIgnorePattern": "^_" }],
+    "vue/no-use-v-if-with-v-for": ["error", {"allowUsingIterationVar": true}]
```

Comment:
is there no way to keep this warning and to locally disable it for the one exceptional usage of this structure.

So we avoid at least to add more of these than necessary

Reply:
it's not possible to disable eslint in template...